apps/console/src/features/server-configurations/constants/server-configurations-constants.ts

```diff
@@ -321,6 +329,10 @@ export class ServerConfigurationsConstants {
 	public static readonly OFFLINE_PASSWORD_RESET: string = "Recovery.AdminPasswordReset.Offline";
 	public static readonly ADMIN_FORCED_PASSWORD_RESET_EXPIRY_TIME: string = "Recovery.AdminPasswordReset.ExpiryTime";
 
+    //multi attribute login set
```

Comment:
The const name is self explanatory IMO
```suggestion
```

Reply:
fixed with https://github.com/wso2/identity-apps/pull/2978/commits/8a276d8dd97d945b5e955fb2c47e010cb23f6dc5